virtual-vms/run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+exec docker run -it --rm -v $PWD/data:/vms qubell/virtual-vms:latest sh
```

Comment:
Well, this is just a convenience tool to test the image, I thought it might be useful. But if you insist I'll remove it.